src/Core/Settings.h

```diff
@@ -514,4 +514,13 @@ struct Settings : public BaseSettings<SettingsTraits>
     static void checkNoSettingNamesAtTopLevel(const Poco::Util::AbstractConfiguration & config, const String & config_path);
 };
 
+/*
+ * User-specified file format settings for File and ULR engines.
```

Comment:
```suggestion
 * User-specified file format settings for File and URL engines.
```

Reply:
Oops, we forgot the typo, I'll fix manually later.